packages/renderer/src/lib/ui/PasswordInput.svelte

```diff
@@ -38,7 +38,7 @@ async function onShowHide() {
   bind:element={element}>
   <svelte:fragment slot="right">
     <button
-      class="px-1 cursor-pointer text-gray-700 group-hover:text-gray-900 group-focus-within:text-gray-900"
+      class="px-1 cursor-pointer text-[var(--pd-content-text)] group-hover:text-[var(--pd-content-sub-header)] group-focus-within:text-[var(--pd-content-sub-header)]"
```

Comment:
PasswordInput extends Input, so these should come from `--pd-input-field-`*. Otherwise someone could customize just inputs to be blue and this button wouldn't match. The values from NumberInput would be a good starting point.

Reply:
fixed!